.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.formatting.provider": "black"
```

Comment:
I think you can add `.vscode/` to the gitignore.

Reply:
Yea sorry, this should be in `.gitignore`.